glitter/pages/middleware.py

```diff
@@ -3,6 +3,9 @@
 from django.http import Http404, HttpResponseRedirect
 
 from glitter.exceptions import GlitterRedirectException, GlitterUnpublishedException
+from django.conf import settings
+from glitter import urls as glitter_urls
+import importlib
```

Comment:
How'd you mean? The module its importing is `importlib`? Want me to do `from importlib import reload`?